ksm/ksm_day7/체육복.js

```diff
@@ -0,0 +1,22 @@
+function solution(n, lost, reserve) {
+    const newLost = lost.filter(cloth => !reserve.includes(cloth)); // lost와 reserve 중복 번호 제거
+    const newReserve = reserve.filter(cloth => !lost.includes(cloth)); // reserve와 lost 중복 번호 제거
+  
+    newLost.sort((a, b) => a - b); 
+    newReserve.sort((a, b) => a - b);
+    
+    let count = n - newLost.length; // 체육복을 잃어버리지 않은 사람 수
+    
+    for (let i = 0; i < newLost.length; i++) {
+        for (let j = 0; j < newReserve.length; j++) {
+            if (newLost[i] === newReserve[j] - 1 || newLost[i] === newReserve[j] + 1) { // 여분 체육복이 있는 번호의 앞, 뒷 번호 사람에게 빌려줌
```

Comment:
```suggestion
        for (let j = 0; j < newReserve.length; j++) {
            // 여분 체육복이 있는 번호의 앞, 뒷 번호 사람에게 빌려줌
            if (newLost[i] === newReserve[j] - 1 || newLost[i] === newReserve[j] + 1) { 
```
주석 길이가 길어서 아래 줄로 내려왔네요. 주석이 우측으로 너무 길게 작성되어 있으면 가독성이 떨어질 수도 있습니다. 
이럴땐 코드 바로 위에 작성하는 방법도 괜찮아보입니다.